kolibri/core/assets/src/views/core-base.vue

```diff
@@ -103,6 +103,8 @@
         return {
           top: `${this.headerHeight}px`,
           [this.isRtl ? 'right' : 'left']: 0,
```

Comment:
Should add in `paddingTop` too!

Reply:
Add `padding-top: 32px` within the css.

Reply:
I think it is reasonable to have the dynamic padding for the top as well, in order to reduce unused whitespace on mobile.